share/initialization/common

```diff
@@ -90,6 +90,8 @@ fi
 
 # some useful shortcuts
 alias -- -='cd -'
+alias h='history 25'
```

Comment:
I should have used `fc` here instead of history because it is a built in. I will fix this.